timesketch/api/v1/resources/sigma.py

```diff
@@ -153,11 +217,371 @@ def post(self):
             )
             abort(
                 HTTP_STATUS_CODE_BAD_REQUEST,
-                "Sigma parsing error: invalid yaml provided {0!s}".format(exception),
+                "Sigma parsing error: invalid yaml provided: {0!s}".format(
+                    exception
+                ),
             )
 
-        if sigma_rule is None:
+        if not sigma_rule:
             abort(HTTP_STATUS_CODE_NOT_FOUND, "No sigma was parsed")
+
+        return jsonify({"objects": [sigma_rule], "meta": {}})
+
+
+class SigmaRuleListResource(resources.ResourceMixin, Resource):
+    """Resource to get list of all SigmaRules."""
+
+    @login_required
+    def get(self):
+        """Fetches Sigma rules from the database
+
+        Fetches fetch all Sigma rules stored in the database on the system
+        and returns a list of JSON representations of the rules.
+
+
+        Returns:
+            List of sigma rules represented in JSON e.g.
+                {"objects": [sigma_rules], "meta": {"rules_count": 42}
+        """
+        sigma_rules = []
+
+        all_sigma_rules = SigmaRule.query.all()
+        for rule in all_sigma_rules:
+            sigma_rules.append(_enrich_sigma_rule_object(rule=rule))
+
+        meta = {"rules_count": len(sigma_rules)}
+        return jsonify({"objects": sigma_rules, "meta": meta})
+
+
+class SigmaRuleResource(resources.ResourceMixin, Resource):
+    """Resource to read / delete / create / update a Sigma rule."""
+
+    @login_required
+    def get(self, rule_uuid):
+        """Fetches a single Sigma rule from the databse.
+
+        Fetches a single Sigma rule selected by the `uuid` in
```

Comment:
```suggestion
        Fetches a single Sigma rule selected by the `UUID` in
```

Please capitalize UUID everywhere in docstrings and comments.